tests/fs/test_memfs_fsync.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
+#include <emscripten.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <sys/stat.h>
+#include <errno.h>
+#include <string.h>
+
+int result = 1;
+
+int main() {
+  int fd;
+
+  // We first make sure the file doesn't currently exist
```

Comment:
This comment is referring to `stat()` call in which I copied from `test_idbfs_syncfs.c`.